app/controllers/api/status_controller.rb

```diff
@@ -35,14 +35,6 @@ def check
     if params[:score]
       fields.push :score
     end
-    render json: @projects.to_json({
-      only: fields,
-      methods: [:package_manager_url, :stars, :forks, :keywords, :latest_download_url],
-      include: {
-        versions: {
-          only: [:number, :published_at]
-        }
-      }
-    })
```

Comment:
what's our test coverage like on this - how confident are we that existing API users will still get the same thing they were getting?

Reply:
Test coverage is very thin and there isn't anything to really check the validity or structure of the response. Are you worried about missing fields or something else? I can add a test to verify all fields are there that we expect, but there could be an issue with the actual data being somewhat different even with that test.

Reply:
I have no idea from reading the patch whether the above code and the serializer produce the same fields in the same structure - I just don't know enough about how they work to know if we might be accidentally changing the API and breaking ourselves. If you inspected some examples manually then that'd be a start for sure.

Reply:
I have compared my local API response to one I get from production to verify all the fields are coming back and it looks good. I added a test that has an array with all the fields we expect to come back and verify the response contains all of them.